conans/tools.py

```diff
@@ -121,7 +121,18 @@ def environment_append(env_vars):
         os.environ.update(old_env)
 
 
-def build_sln_command(settings, sln_path, targets=None, upgrade_project=True):
+def msvc_command(settings, sln_path, targets=None, upgrade_project=True, build_type=None,
+                 arch=None):
+    """ Do both: set the environment variables and call the .sln build
+    """
+    vcvars = vcvars_command(settings)
```

Comment:
I would prefer `build_sln_with_vcvars_command` or similar. I know that it's long but better long but meaninful.